internal/provider/resource_machine_test.go

```diff
@@ -59,8 +94,12 @@ func TestAcc_ResourceMachine_Minimal(t *testing.T) {
 			},
 			{
 				ImportStateVerify: true,
-				ImportState:       true,
-				ResourceName:      resourceName,
+				// We ignore the hostname and wait_for_hostname during ImportVerify in our tests
+				// we don't wait for the hostname to be populated, however during the import
+				// we set it.
```

Comment:
I'm a bit confused reading this. "During import we set it", can we reword this/make it more explicit?